libs/article/website/src/lib/article-list/article-list-item.tsx

```diff
@@ -0,0 +1,58 @@
+import {Article} from '@wepublish/website/api'
+import {format as formatDate} from 'date-fns'
+import {Theme, styled, css, useTheme} from '@mui/material'
+import {useWebsiteBuilder} from '@wepublish/website/builder'
+
+export const ArticleListItemImageWrapper = styled('div')`
+  display: grid;
+  width: 100%;
+  min-height: 450px;
+`
+
+export const ArticleListItemContent = styled('div')``
+
+export const wrapperStyles = (theme: Theme) => css`
+  display: grid;
+  gap: ${theme.spacing(2)};
+  text-align: left;
+  text-decoration: none;
+  color: inherit;
+  align-items: stretch;
+
+  ${theme.breakpoints.up('sm')} {
```

Comment:
The size of the element should come from the parent, children should not know about layouts.